host_guest/WP6/README.md

```diff
@@ -0,0 +1,37 @@
+#
+The SAMPL9 WP6 challenge
+
+The WP6 challenge focuses on binding of WP6 to thirteen ammonium/diammonium cationic guests. Experimental binding measurements have been collected and the Isaacs group is preparing a paper for publication. Experimental details will be provided shortly. 
+
+Our deadline for this challenge will be posted in the top-level [README.md](https://github.com/samplchallenges/SAMPL9/blob/master/README.MD#the-wp6-challenge)
+
+If you want any announcement of updates/changes/fixes here, [sign up for our e-mail list](INSERT NEW SIGN UP LINK HERE)
+
+## A quick view of the host and guests
+
+![](images/WP6_overview.png)
+
+Please be sure to read the Disclaimers section below to note poitns which may require particular attention
+
+## More on the host
+
+This host-guest series is based on a pillar[n]arene host known as WP6, yet to be featured in a previous SAMPL challenge. WP6 similar to cucurbit[n]urils, application based. Structurally, WP6 contains CH2 linkers connecting the phenylene groups, and each phenylene group contains two anionic functional group arms. Background information on WP6 may be found in a number of publications including DOI [insert DOIs HERE]
+
+## Experimental conditions
+
+All ITC experiments were conducted in phosphate buffered saline (1x PBS) at pH 7.40 and at 298.15 K. [NEED TO INPUT CONCENTRATION]
```

Comment:
The way I would deal with outstanding items is to not put this in for now, and then raise an issue listing things that you will clarify. Then wehn we get the info from Isaacs we can add it. If you put in placeholders like this we can't merge until we have the info to fill them in.